apps/desktop/src/trpc/routers/models.ts

```
@@ -149,6 +243,304 @@
+      );
+      if (!modelManagerService) {
+        throw new Error("Model manager service not initialized");
+      }
+
+      switch (input.type) {
+        case "speech":
+          await modelManagerService.setSelectedModel(input.modelId);
+          // Notify transcription service about model change
+          const transcriptionService = ctx.serviceManager.getService(
+            "transcriptionService",
+          );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)